src/party.gleam

```diff
@@ -75,6 +75,21 @@ pub fn satisfy(when pred: fn(String) -> Bool) -> Parser(String, e) {
   })
 }
 
+/// Parse a single character.
```

Comment:
Is there a reason you've written it this way instead of just ```
pub fn any_char() -> Parser(String, e) {
  satisfy(fn(_c) { True })
}
```

Reply:
Thought the other one would be slightly more performant because it skips a `case` expression
But yeah... that one is more ergonomic